packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -520,6 +532,14 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     return <BaseButton {...splitButtonProps} onMouseDown={ this._onMouseDown } tabIndex={ -1 } />;
   }
 
+  private _onSplitButtonClick = (ev: React.MouseEvent<HTMLDivElement | HTMLAnchorElement>) => {
```

Comment:
This method isn't used anywhere, how is it getting called?

Reply:
You're right, it's not this was old code that came from a merge conflict. I'll get rid of it.